app/javascript/packs/banner_editor.jsx

```diff
@@ -425,13 +446,14 @@ export default class BannerEditor extends React.Component {
     // Three POST requests make up the save: Content, Logo, and Cover.
     this.setState({state: 'save', saving: 'true'}, () => Spinner.show('save-spinner', 'save-container'))
 
-    let url = '/publishers/' + document.getElementById("publisher_id").value + "/site_banners";
+    let url = '/publishers/' + document.getElementById("publisher_id").value + "/site_banners/save?channel_id=" + this.props.channels[this.state.channelIndex].id;
```

Comment:
This is where I think there should be logic in place to check to see if the channelId exists, if it doesn't then we run `/publishers/:id/site_banners/create` without a channel id. If there's not a channel id then the controller knows that the user is trying to create a default channel